builds/e2e/templates/e2e-setup.yaml

```diff
@@ -77,6 +82,35 @@ steps:
     Write-Output "##vso[task.setvariable variable=binDir]$binDir"
   displayName: Build tests
 
+- pwsh: |
+    $manifestSignerClientDir = '$(Build.SourcesDirectory)/samples/dotnet/ManifestSignerClient'
+    dotnet build $manifestSignerClientDir
+    $manifestSignerClientBinDir = Convert-Path "$manifestSignerClientDir/bin/Debug/netcoreapp3.1"
+    Write-Output "##vso[task.setvariable variable=manifestSignerClientDir]$manifestSignerClientDir"
+    Write-Output "##vso[task.setvariable variable=manifestSignerClientBinDir]$manifestSignerClientBinDir"
+    $manifestSigningTestDir = '$(System.ArtifactsDirectory)/manifest_signing'
+    $manifestSigningDeploymentDir = '$(System.ArtifactsDirectory)/manifest_signing/deployment'
+    Write-Output "##vso[task.setvariable variable=manifestSigningDeploymentDir]$manifestSigningDeploymentDir"
+    $manifestSigningCertDir = '$(System.ArtifactsDirectory)/manifest_signing/certs'
+    New-Item "$manifestSigningCertDir" -ItemType Directory -Force | Out-Null
+    New-Item "$manifestSigningDeploymentDir" -ItemType Directory -Force | Out-Null
+    New-Item -Path "$manifestSigningDeploymentDir/deployment.json" -ItemType File
```

Comment:
Do we need to create the deployment.json and signed_deployment.json file here?  This doesn't seem to be hurting anything, but is it necessary?